app/views/questionnaires/_questionnaire_debug.html.slim

```diff
@@ -1,4 +1,4 @@
 - if ENV['RAILS_ENV'].eql?('development')
   pre.debug_dump
-    = @questionnaire.debug_summary
```

Comment:
I wrote this but never committed it 🤷🏼‍♂️ . Instead can you stub the method on `Questionnaire` because I think we should add it.